test/test_indexing.py

```diff
@@ -67,21 +61,18 @@ def test_multiple_bool_indices(self, device):
     def test_byte_mask(self, device):
         v = torch.randn(5, 7, 3, device=device)
         mask = torch.ByteTensor([1, 0, 1, 1, 0]).to(device)
-        with warnings.catch_warnings(record=True) as w:
-            self.assertEqual(v[mask].shape, (3, 7, 3))
-            self.assertEqual(v[mask], torch.stack([v[0], v[2], v[3]]))
-            self.assertEquals(len(w), 2)
+        with self.assertRaisesRegex(
```

Comment:
nit: since you are fixing the tests, mask can also be created using non-deprecated API `a=torch.tensor([1,0,1,1,0], device="cpu", dtype=torch.uint8)   `